game-core/src/main/java/games/strategy/engine/framework/startup/launcher/LocalLauncher.java

```diff
@@ -61,4 +43,21 @@ protected void launchInNewThread(final Component parent) {
       SwingUtilities.invokeLater(() -> JOptionPane.getFrameForComponent(parent).setVisible(true));
     }
   }
+
+  private Optional<ServerGame> loadGame() {
+    try {
+      gameData.doPreGameStartDataModifications(playerListing);
+      final Messengers messengers = new Messengers(new HeadlessServerMessenger());
+      final Set<IGamePlayer> gamePlayers = gameData.getGameLoader().createPlayers(playerListing.getLocalPlayerTypes());
+      final ServerGame game = new ServerGame(gameData, gamePlayers, new HashMap<>(), messengers);
+      game.setRandomSource(randomSource);
+      gameData.getGameLoader().startGame(game, gamePlayers, headless);
+      return Optional.of(game);
+    } catch (final Exception ex) {
+      ClientLogger.logError("Failed to start game", ex);
```

Comment:
This change in logging is intentional. I don't think we should sort-of-swallow this potential error, especially because if an error occurs, the game is silently not being started at all.